src/sentry/db/postgres/base.py

```diff
@@ -29,6 +29,8 @@ def remove_null(value):
     # somewhat legible rather than error. Considering this is better
     # behavior than the database truncating, seems good to do this
     # rather than attempting to sanitize all data inputs now manually.
+    if type(value) is bytes:
+        return value.replace(b"\x00", b"")
```

Comment:
The goal wasn't to remove it, the goal was to preserve it.

The original behavior caused the text to get truncated in C because of null terminated strings. So not sure we'd wanna regress and remove the value if we can avoid it.

Reply:
it looks like something changed in here recently to where it's now just removed.

Reply:
Here's where it was changed https://github.com/getsentry/sentry/pull/10752

Reply:
There is no way to preserve it, the change here looks correct. It's just the path for bytes that was missing for py3.